xlsynth/src/sv_bridge_builder.rs

```diff
@@ -249,15 +292,15 @@ mod tests {
     fn test_convert_leaf_module_struct_def_only() {
         let dslx = r#"
         struct MyStruct {
-            byte_data: u8,
+            byte_data: u8[10],
             word_data: u16,
         }
         "#;
         let sv = simple_convert_for_test(dslx).unwrap();
         assert_eq!(
             sv,
             r#"typedef struct packed {
-    logic [7:0] byte_data;
+    logic [9:0] [7:0] byte_data;
```

Comment:
This one doesn't look right but we can land and iterate on this.

Reply:
Changed the name to reflect the new definition it is a byte_array now.